pkg/apiserver-impl/starterserver.go

```diff
@@ -129,7 +130,11 @@ func StartServer(
 		cancelFunc()
 	}
 
-	klog.V(0).Infof("API Server started at localhost:%d/api/v1", listeningPort)
+	if feature.IsEnabled(ctx, feature.UIServer) {
+		log.Spinner(fmt.Sprintf("Web console accessible at http://localhost:%d/", listeningPort)).End(true)
```

Comment:
Can we make all these messages (or just the URLs) stand out, by printing them in bold green for example (just like the info about ports forwarded)?

Reply:
I'm not sure. URLs are already clickable and highlighted in some way by terminals, as they start with http://

Reply:
I guess it would depend on the terminal emulator, because mine only highlights the link when I hover over it. But I was mostly concerned about having a way to quickly visually catch the information, in case it is lost in the middle of the logs (for example if I run `odo dev -v 2` or even just `odo dev --logs`).
Actually, what do you think of having all this displayed under the "Dev mode" status info?

```
[...]
↪ Dev mode
 Status:
 Watching for changes in the current directory /home/asoro/work/tmp/6963-move-odo-dev-ui-out-of-the-experimental-mode
 - Web console accessible at http://localhost:20000/
 - API Server started at http://localhost:20000/api/v1
 - API documentation accessible at http://localhost:20000/swagger-ui/

 Keyboard Commands:
[Ctrl+c] - Exit and delete resources from the cluster
     [p] - Manually apply local changes to the application on the cluster

```

Reply:
The info can be difficult to find with `-v 2`, but is it a standard way to run `odo dev`? Without this flag, it appears at the very beginning of the output, it seems to me very easy to find (even when `--logs` is used)

Why not for the web console, but for the API server/doc, these URLs will be used only by plugins developers, I don't think we want to spam the user with this info.

Reply:
I have made the change to display the Web UI URL with the Status